hack/create-operatorhub-prs.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/sh
+# Copyright 2023 Red Hat, Inc. and/or its affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+VERSION=$1
+if [[ $1 == v* ]]; then TAG=$1; else TAG=v$1; fi
+COMMUNITY_OPERATORS_DIR=community-operators
+COMMUNITY_OPERATORS_REPO=$2
+COMMUNITY_OPERATORS_PROD_DIR=community-operators-prod
+COMMUNITY_OPERATORS_PROD_REPO=$3
+if [[ $4 == false ]]; then DRY_RUN=false; else DRY_RUN=true; fi
+
+git fetch --tags --all
+echo "Checking out Kogito $TAG"
+git checkout tags/$TAG -B kogito-$TAG
+
+cd ../
```

Comment:
what is the purpose here ?

Reply:
Checking out the other 2 repos not in a child directory of the kogito-operator one in order to have these 3 folders:
- kogito-operator
- community-operators
- community-operators-prod

Reply:
I would say that this is too specific to the environment

Reply:
we should be able to run it from any kogito-operator repository, without having to perform any manual operations. wdyt ?

Reply:
Ok, I understood your point. Even if we are able to run it into a GHA also without any change, I started working into the ``tmp`` folder for checking out the the 2 repos we need to change.

I also added an additional optional variable for in order to make also the Kogito Operator directory customizable (I don't think we need it when we use the ``Makefile``)